source/operations/cert-manager.rst

```diff
@@ -0,0 +1,108 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
+--------------------------------------------
+
+This guide shows you how to install `cert-manager <https://cert-manager.io>`__ and set up the global ``Cluster Issuer``.
+
+cert-manager manages certificates within Kubernetes clusters.
+The MinIO Operator supports using cert-manager for certificates as an alternative to the MinIO Operator managing certificates for itself and its tenants.
+
+cert-manager obtains valid certificates from an ``Issuer`` or ``ClusterIssuer`` and can automatically renew certificates prior to expiration.
+
+A ``ClusterIssuer`` issues certificates for multiple namespaces.
+An ``Issuer`` only mints certificates for its own namespace.
+
+The following graphic depicts how various namespaces make use of either an ``Issuer`` or ``ClusterIssuer`` type.
+
+- cert-manager is installed in the ``cert-manager`` namespace, which does not have either an ``issuer`` or a ``ClusterIssuer``.
```

Comment:
Do we still need this if we're dropping a reference to it in the diagram?

Maybe:

```
The following diagram provides a high level view of the relationship between Cluster Issuers and Issuers in a cert-manager context:

Reply:
Holding for the new diagram. Will leave unresolved until then.